### PR TITLE
ACL token passing integration test

### PR DESCRIPTION
**Pull request by chemicL**
The ACL token passing has been carried over from the forked project, but it had no tests.
I added an integration test, which proves the functionality works as expected.

### CODE REVIEW

**Comment by janisz** [comment]
```
--- FAIL: TestACLTokenPassing (2.06s)
	Error Trace:	consul_test.go:1106
	Error:		"[%!s(*service.Service=&{serviceA.0_serviceA_8080 serviceA [marathon test marathon-task:serviceA.0] 127.0.0.1 true})]" should have 0 item(s), but has 1
	Messages:	Reading services list without ACL token should yield empty response
```
https://travis-ci.org/allegro/marathon-consul/builds/289041586#L2073

File: consul/consul_test_server.go [hunk 1]
```diff
@@ -32,6 +32,32 @@ func CreateTestServer(t *testing.T) *testutil.TestServer {
...
 
+const MasterToken = "masterToken"
+
+func CreateSecuredTestServer(t *testing.T) *testutil.TestServer {
+	ports, err := getPorts(6)
+	assert.NoError(t, err)
+
+	server, err := testutil.NewTestServerConfig(func(c *testutil.TestServerConfig) {
+		c.Datacenter = fmt.Sprint("dc-", time.Now().UnixNano())
+		c.Ports = &testutil.TestPortConfig{
```
Comment (janisz) [inline consul/consul_test_server.go]: Can you extract this to a fucntion that returns `*testutil.TestPortConfig`

File: consul/consul_test.go [hunk 2]
```diff
@@ -1057,3 +1057,51 @@ func Test_substituteEnvironment(t *testing.T) {
 		})
 	}
 }
+
+func TestACLTokenPassing(t *testing.T) {
```
Comment (janisz) [inline consul/consul_test.go]: Change name of this tests to define what happens here. e.g., `TestIfTokenIsUsedToConnectToConsul`

Comment (chemicL) [reply consul/consul_test.go]: Changed to TestTokenIsUsedToConnectToConsul

File: consul/consul_test_server.go (hunk 1 above)
Comment (chemicL) [reply consul/consul_test_server.go]: done

**Comment by janisz** [comment]
@chemicL  Thanks a lot

**Pull request merged**